ci/run-sanity.sh

```diff
@@ -56,14 +56,26 @@ $agave_validator --ledger config/ledger exit --force || true
 
 wait $pid
 
-$solana_ledger_tool create-snapshot --ledger config/ledger "$snapshot_slot" config/snapshot-ledger
-cp config/ledger/genesis.tar.bz2 config/snapshot-ledger
-$solana_ledger_tool copy --ledger config/ledger \
-  --target-db config/snapshot-ledger --starting-slot "$snapshot_slot" --ending-slot "$latest_slot"
-$solana_ledger_tool verify --abort-on-invalid-block \
-  --ledger config/snapshot-ledger --block-verification-method blockstore-processor
-$solana_ledger_tool verify --abort-on-invalid-block \
-  --ledger config/snapshot-ledger --block-verification-method unified-scheduler
+for method in blockstore-processor unified-scheduler
+do
+  rm -rf config/snapshot-ledger
+  $solana_ledger_tool create-snapshot --ledger config/ledger "$snapshot_slot" config/snapshot-ledger
+  cp config/ledger/genesis.tar.bz2 config/snapshot-ledger
+  $solana_ledger_tool copy --ledger config/ledger \
+    --target-db config/snapshot-ledger --starting-slot "$snapshot_slot" --ending-slot "$latest_slot"
+
+  set -x
+  $solana_ledger_tool --ledger config/snapshot-ledger slot "$latest_slot" --verbose --verbose \
+    |& grep -q "Log Messages:$" && exit 1
```

Comment:
can you explain this? and why we lost `--abort-on-invalid-block`?

Reply:
this is the `slot` subcommand of `ledger-tool`. and it won't accept the flag...

this `slot` invocation and the later one is testing `Log Messages:` will start to appear only after the `verify` subcommand.